spec/features/publish/courses/editing_subject_spec.rb

```diff
@@ -13,30 +13,69 @@
     when_i_select_a_primary_subject('Primary with English')
     and_i_click_continue
     then_i_am_met_with_course_details_page
-    and_i_should_see_a_success_message
+    and_i_should_see_subject_updated_success_message
   end
 
   scenario 'updating secondary subject' do
     and_there_is_a_secondary_course_i_want_to_edit
     when_i_visit_the_edit_course_subject_page
-    when_i_select_a_subject(:business_studies)
+    when_i_select_a_master_subject(:business_studies)
     and_i_click_continue
     then_i_am_met_with_course_details_page
-    and_i_should_see_a_success_message
+    and_i_should_see_subject_updated_success_message
+  end
+
+  scenario 'updating a course with two secondary subjects' do
+    and_there_is_a_secondary_course_i_want_to_edit
+    and_the_course_has_two_subjects
+    when_i_visit_the_edit_course_subject_page
+    then_i_should_see_populated_selects('Drama', 'Latin')
```

Comment:
```suggestion
    then_i_see_populated_selects('Drama', 'Latin')
```

I won't mark them all, but just to keep us from having to change this later if we introduce the rubocop rule that prohibits it, let's get rid of the 'should's in the tests

Reply:
I agree, and had to restrain myself from changing them initially.